src/components/visualizations/Maps/useMap.ts

```diff
@@ -4,10 +4,12 @@ import maplibregl, { type LngLatBoundsLike } from 'maplibre-gl';
 // Note (AM): This needs to be scoped in MapTiler or hidden with a Secret Manager.
 const API_KEY = 'VStCFFYMJAABHpPVId3w';
 
-export function useMap(bounds: LngLatBoundsLike = [
-        [-71.5, 41.92],
-        [-71.16, 41.32],
-      ]) {
+export function useMap(
+  bounds: LngLatBoundsLike = [
+    [-71.5, 41.92],
+    [-71.16, 41.32],
+  ]
+) {
```

Comment:
If we're ever putting `bounds` in a dependency array, I think we should define it outside the function parameters. Otherwise, every component rerender will re-call `useMap` and instantiate a new variable `bounds` and cause the things in that dependency to be re-run. 

```suggestion
const BOUNDS: LngLatBoundsLike = [
    [-71.5, 41.92],
    [-71.16, 41.32],
];
export function useMap(bounds: LngLatBoundsLike = BOUNDS) {
```